### PR TITLE
Add non confirming processors

### PR DESCRIPTION
**Pull request by aljones15**
- asserts on 400 or 422
- moves a test data gen from suite to setup
- adds missing normative statement

### CODE REVIEW

File: tests/assertions.js [hunk 1]
```diff
@@ -30,10 +30,7 @@ export const verificationFail = async ({
   should.not.exist(result, reason);
   should.exist(error, 'Expected verifier to error.');
   should.exist(error.status, 'Expected verifier to return an HTTP Status code');
-  error.status.should.equal(
-    400,
-    'Expected HTTP Status code 400 invalid input!'
-  );
+  error.status.should.be.oneOf([400, 422], reason);
```
Comment (BigBlueHat) [inline tests/assertions.js:33]: Should this `reason` be something closer to the original rather than "Expected no result from verifier."?

**Review by BigBlueHat — APPROVED** [review]
Mostly just a question. Otherwise, looks good.

File: tests/assertions.js (hunk 1 above)
Comment (aljones15) [reply tests/assertions.js:33]: hmmmm... I honestly was just thinking about the single case here where Greg's implementation runs 500 and I didn't know which test vector was causing that, so I think having a different default makes sense and I'll make that change momentarily. good idea.

Comment (aljones15) [reply tests/assertions.js]: added the additional default here https://github.com/w3c/vc-di-bbs-test-suite/pull/49/commits/f1fbfad829c6221eb394380662ee0468b59cbf5a

**Comment by TallTed** [comment]
This should be retitled to `Add non-conforming processors`. (`confirming` is an typo.)

**Pull request merged**